src/main/treeDataProviders/dependenciesTree/dependenciesRoot/npmTree.ts

```diff
@@ -23,17 +23,25 @@ export class NpmTreeNode extends RootNode {
             projectDetails.loadProjectDetails(NpmCmd.runNpmLs(this.workspaceFolder));
         } catch (error) {
             this._logManager.logError(<any>error, false);
+            projectDetails.loadProjectDetails(JSON.parse((<any>error).stdout.toString()));
             projectDetails.loadProjectDetailsFromFile(path.join(this.fullPath));
             npmLsFailed = true;
         }
         this.populateDependenciesTree(this, projectDetails.dependencies);
         if (npmLsFailed) {
-            this.topSeverity = Severity.Unknown;
-            this.buildError = BuildTreeErrorType.NotInstalled;
-            this._logManager.logMessageAndToastErr(
-                `Failed to scan npm project. Hint: Please make sure the commands 'npm install' or 'npm ci' run successfully in '${this.workspaceFolder}'`,
-                'ERR'
-            );
+            if (this.children.length === 0) {
+                this.topSeverity = Severity.Unknown;
+                this.buildError = BuildTreeErrorType.NotInstalled;
+                this._logManager.logMessageAndToastErr(
+                    `Failed to scan npm project. Hint: Please make sure the commands 'npm install' or 'npm ci' run successfully in '${this.workspaceFolder}'`,
+                    'ERR'
+                );
+            } else {
+                this._logManager.logMessageAndToastErr(
```

Comment:
I think toasting is too aggressive here. There are projects that have errors but still wish to utilize this plugin without addressing their errors.
```suggestion
                this._logManager.logMessage(
```